### PR TITLE
Add docs for setting http status code in functions

### PR DESCRIPTION
**Pull request by jonbretman**

### CODE REVIEW

**Review by davenewza — APPROVED** [review]

File: pages/functions.mdx [hunk 1]
```diff
@@ -402,6 +409,47 @@ export default MyFunction((inputs, api, ctx) => {
 });
 ```
 
+## HTTP Status Codes
+
+You can set the HTTP status code of the response from your function, which is useful if you want to return a redirect response from your action.
```
Comment (davenewza) [inline pages/functions.mdx:414]: Returning a redirect is one example.  I think another good example is a webhook expecting some specific status code (i.e. 201 for success and 4xx for error)

File: pages/functions.mdx [hunk 2]
```diff
@@ -402,6 +409,47 @@ export default MyFunction((inputs, api, ctx) => {
...
+return null;
+});
+
+````
+  </Tab>
+   <Tab>
+```keel filename="schema.keel"
+model MyModel {
+  actions {
+    write myRedirectFunction(Any) returns (Any) @function
```
Comment (davenewza) [inline pages/functions.mdx:438]:
```suggestion
    write myRedirectFunction(Any) returns (Any)
```

File: pages/functions.mdx [hunk 3]
```diff
@@ -402,6 +409,47 @@ export default MyFunction((inputs, api, ctx) => {
...
+```tsx filename="functions/myRedirectFunction.ts"
+import {MyRedirectFunction} from "@teamkeel/sdk"
+
+export default MyRedirectFunction((inputs, api, ctx) => {
+  // do some stuff...
+
+// return a redirect
+ctx.response.headers.set("Location", "https://some.url.com/");
+ctx.response.status = 302;
+return null;
```
Comment (davenewza) [inline pages/functions.mdx:429]:
```suggestion
  // return a redirect
  ctx.response.headers.set("Location", "https://some.url.com/");
  ctx.response.status = 302;
  return null;
```

File: pages/functions.mdx [hunk 4]
```diff
@@ -402,6 +409,47 @@ export default MyFunction((inputs, api, ctx) => {
...
+## HTTP Status Codes
+
+You can set the HTTP status code of the response from your function, which is useful if you want to return a redirect response from your action.
+
+import { Tab, Tabs } from "nextra-theme-docs";
+
+<Tabs items={['Function', 'Schema']}>
+  <Tab>
+```tsx filename="functions/myRedirectFunction.ts"
+import {MyRedirectFunction} from "@teamkeel/sdk"
```
Comment (davenewza) [inline pages/functions.mdx:421]:
```suggestion
import { MyRedirectFunction } from "@teamkeel/sdk"
```

File: pages/functions.mdx (hunk 2 above)
Comment (jonbretman) [reply pages/functions.mdx:438]: You don't need `@function` for read/write do you?

Comment (davenewza) [reply pages/functions.mdx:438]: Nope.  I'm wondering now what will happen if you do provide it.  Will `keel generate` provide an implemention :O

File: pages/functions.mdx (hunk 1 above)
Comment (jonbretman) [reply pages/functions.mdx:414]: Yeh I think more examples would be good, but this page is getting a bit long now and so I think needs breaking up. For now lets keep it simple.

**Pull request merged**